Adyen/Overview/02_Features.md

```diff
@@ -0,0 +1,16 @@
+# Key features
+
```

Comment:
- Interpunktion (am Ende des Satzes) konsistent machen, wenn möglich
- In anderen Verbindungen Abschnitt "How it works" (So funktioniert es auf der Webseite). Hier auch zur Konsistenz?
z.B. https://github.com/Actindo-AG/Documentation/blob/main/Shopware6/Overview/02_Features.md
https://github.com/Actindo-AG/Documentation/blob/main/CommerceCloud/Overview/02_Features.md